### DISCUSSION TITLE
Github pages does not deploy to /

### DISCUSSION THREAD

**Discussion by rthomazel** [community/community#137526] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

I'm trying to get `https://github.com/tcodes0/tcodes0` deployed to the root (/) of www.golang.dev.br a domain I own. This used to work and there's an old version of the site live there, but updates have been deploying to http://www.golang.dev.br/tcodes0 not /. The settings do show "main branch - deploy to root (/)". After looking at [documentation](https://docs.github.com/en/pages/configuring-a-custom-domain-for-your-github-pages-site/managing-a-custom-domain-for-your-github-pages-site?platform=linux) it looks like re-adding the custom domain would fix it but I have been unsuccessful in that attempt. In repo settings > pages it either shows a disabled button or I receive an error "this domain is already used by a repo in your account" when attempting removal. Also tried [a few things](https://github.com/orgs/community/discussions/22506) but no luck. 

tried:

added a blank jekyll theme
deleting the CNAME manually (I used to use gh pages without a CNAME file)
adding a CNAME to another domain
adding a CNAME to my domain and deleting it

**Comment by ThisIs-Developer** [comment]
### Fixing GitHub Pages Deployment Issue

1. **Verify GitHub Pages Settings:**
   - Ensure **Branch** is set to `main` and **Folder** is set to `/` in **Settings** > **Pages**.

2. **Check Custom Domain:**
   - Confirm the domain (`www.golang.dev.br`) is correctly set.
   - If errors occur, clear browser cache or use incognito mode.

3. **Remove and Re-add Domain:**
   - Try removing and re-adding the custom domain in **Pages** settings.
   - Ensure DNS records are correctly configured for GitHub Pages.

4. **Check DNS Configuration:**
   - Verify DNS settings, including A records or CNAME pointing to GitHub Pages.

5. **Check Existing Deployments:**
   - Ensure there are no conflicting deployments in the `/tcodes0` subdirectory.

### If this is helpful, please mark as answer!😪

**Comment by rthomazel** [comment]
1 - done
2 - no errors
3 - unable to remove and re-add see opening post.
4 - DNS works as github deploys to /tcodes0
5 - no indication of deploy conflicts, but doesn't deploy to /

**Reply by ThisIs-Developer** [reply]
Hi @tcodes0 , Thanks for the update. Since you've already gone through the basic steps and still face issues with removing and re-adding the custom domain, here's what you can try next:

1. **Manually Remove Domain**: Delete the `CNAME` file in your repo's root directory and try re-adding the domain in Pages settings.
   
2. **Check for Domain Conflicts**: Ensure no other repos in your account are using the same domain.

3. **Contact GitHub Support**: If the issue persists, reach out to [GitHub Support](https://support.github.com/request) for help with clearing the domain from their end.

#### [Pages - GitHub Support](https://support.github.com/request/pages)

**Reply by rthomazel** [reply, marked as answer]
I had a forgotten repository in my account that had the domain added to it. Once I removed it from that domain, I was able to add it to the repo I wanted and deploy to root as expected.
